src/Core/Migration/V6_8/Migration1756068709FixCustomerAddressNameFieldLengths.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_8;
```

Comment:
Migrations in the 6_8 namespace will only be executed on 6.8 major line, so they won't be executed for now

Put the migration in the 6.7 namespace and it should be executed automatically